assets/swagger.json

```diff
@@ -6395,6 +6395,13 @@
             "type": "string"
           }
         },
+        "components": {
+          "type": "array",
+          "title": "Components specifies a list of kustomize components to add to the kustmization before building",
```

Comment:
`kustmization` -> kustomization? The same in other files like application-crd.yaml, applicationset-crd.yaml, etc.

Reply:
thank you, good catch!

Reply:
@chengfang fixed